lib/rubocop/cop/rspec/predicate.rb

```diff
@@ -0,0 +1,182 @@
+# encoding: utf-8
+
+module RuboCop
+  module Cop
+    module RSpec
+      # Prefer using predicate matcher instead of truthiness
+      #
+      # @example
+      #   # bad
+      #     expect(foo.valid?).to be_truthy
+      #
+      #   # good
+      #     expect(foo).to be_valid
+      class Predicate < Cop
+        include ConfigurableEnforcedStyle
+
+        MSG = 'Use predicate matcher `%s`'.freeze
+
+        def on_send(node)
+          expectation = correction(node)
+          return if expectation.nil?
+
+          add_offense(
+            node,
+            :expression,
+            format(MSG, expectation)
+          )
+        end
+
+        def autocorrect(node)
+          lambda do |corrector|
+            corrector.replace(node.loc.expression, correction(node))
+          end
+        end
+
+        private
+
+        def requires_predicate?
+          style == :predicate
```

Comment:
Can you write this as `style.equal?(:predicate)`. This is more strict (it signals that `style` will ONLY be that symbol)